internal/pkg/service/buffer/store/model/column/column_test.go

```diff
@@ -114,25 +114,27 @@ func TestColumn_Header(t *testing.T) {
 func TestColumn_Template_Body(t *testing.T) {
 	t.Parallel()
 
-	c := column.Template{Language: column.TemplateLanguageJsonnet, Content: "body.key1"}
+	c := column.Template{Language: column.TemplateLanguageJsonnet, Content: "Body('key1.key2')"}
 
+	val1 := orderedmap.New()
+	val1.Set("key2", "val2")
 	body := orderedmap.New()
-	body.Set("key1", "val1")
-	body.Set("key2", "val2")
+	body.Set("key1", val1)
+	body.Set("key3", "val3")
 
 	header := http.Header{}
 	header.Set("Content-Type", "application/json")
 	header.Set("Content-Encoding", "gzip")
 
 	val, err := c.CsvValue(column.ImportCtx{Body: body, Header: header})
 	assert.NoError(t, err)
-	assert.Equal(t, "\"val1\"\n", val)
+	assert.Equal(t, "\"val2\"\n", val)
 }
 
 func TestColumn_Template_Headers(t *testing.T) {
 	t.Parallel()
 
-	c := column.Template{Language: column.TemplateLanguageJsonnet, Content: `headers['Content-Encoding']`}
+	c := column.Template{Language: column.TemplateLanguageJsonnet, Content: `Headers('Content-Encoding')`}
```

Comment:
Niekde by bolo dobre skusit nastavit hlavicku s inym case: `content-type`, `CONTENT-TYPE`, myslim ze to riesi Go a je jedno ako to je definovane v HTTP hlavicke + je jedno ako to zadas do `Get`, ale male by sme to mat otestovane.

Reply:
HTTP hlavicky su totiz case insensitive

Reply:
https://github.com/keboola/keboola-as-code/pull/1031/commits/84f21dbd2be7288bc392e424a567fd1de806f31f